.semaphore/semaphore.yml

```
@@ -30,26 +27,20 @@ blocks:
           commands:
             - checkout
             - docker-compose up -d muh_android
-            - 'docker-compose exec muh_android ./gradlew lint'
-        - name: bundle
+            - docker-compose exec muh_android ./gradlew lint
+        - name: bundle & dependencies
           commands:
             - checkout
             - docker-compose up -d muh_android
-            - 'docker-compose exec muh_android ./gradlew bundle'
-        - name: dependencies
-          commands:
-            - checkout
+            - docker-compose exec muh_android ./gradlew bundle
             - docker-compose up -d muh_android
-            - 'docker-compose exec muh_android ./gradlew androidDependencies'
-        - name: ktlint
+            - docker-compose exec muh_android ./gradlew androidDependencies
+        - name: ktlint & unit test
           commands:
             - checkout
             - docker-compose up -d muh_android
-            - "docker-compose exec muh_android curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint"
-            - "docker-compose exec muh_android chmod a+x ktlint"
-            - "docker-compose exec muh_android ./ktlint"
-        - name: unit test
-          commands:
-            - checkout
+            - 'docker-compose exec muh_android curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint'
```

Precisa das aspas? 👀 
```suggestion
            - docker-compose exec muh_android curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint
```

---

nope